src/components/invite-dialog/container.test.tsx

```diff
@@ -12,6 +12,8 @@ describe('Container', () => {
       inviteCode: '',
       inviteUrl: '',
       assetPath: '',
+      invitesUsed: 0,
+      maxInvitesPerUser: 0,
```

Comment:
I might rename this to `maxUses` or something slightly more generic. You never know when the intent may change.

Reply:
makes sense. done